test-case-reporting/src/db_interfacing.ts

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
Let's name this file to reflect the class it defines, ie. `test_results_records.ts`

Reply:
Done.